apps/videos-admin/src/components/FormTextField/FormTextField.stories.tsx

```
@@ -0,0 +1,82 @@
+
+export const Disabled: Story = {
+  args: {
+    ...Filled.args,
+    disabled: true,
+    initialValues: {
+      textfield: 'Disabled'
+    }
+  }
+}
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)